site/docs/components/country-symbol/examples.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Country Symbol
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="country-symbol" exampleName="BasicUsage">
+    
+    ### Basic Usage
+
+    Each country symbol should be imported and used individually.
+
+  </LivePreview>
+  <LivePreview componentName="country-symbol" exampleName="Size">
+
+    ### Size
+
+    To change the size of a country symbol, use the `size` prop to multiply the base size.
+
+  </LivePreview>
+  <LivePreview componentName="country-symbol" exampleName="AllCountrySymbols">
+
+    ### All Country Symbols
+
+    The following example shows all the available country symbols. Each component can be imported individually and is named via its country code.
+
+    For example the country symbol for Belgium has the code BE and can be imported as follows:
+
+    ```jsx
+    import { BE } from '@salt-ds/countries;'
+    ```
+
+  </LivePreview>
+  <LivePreview componentName="country-symbol" exampleName="LazyLoading">
+
+    ### Lazy Loading
+
+    For situations like a list where you may be importing many country symbol components at once, you may not want all the components to be included in your initial bundle size. To avoid this you can lazily load the Country Symbol components using [React Suspense](https://react.dev/reference/react/Suspense), alongside the `LazyCountrySymbol` component. LazyCountrySymbol takes a `code` prop to load the appropriate Country Symbol component.
```

Comment:
nit 
```suggestion
    For situations like a list where you may be importing many country symbol components at once, you may not want all the components to be included in your initial bundle size. To avoid this you can lazily load the Country Symbol components using [React Suspense](https://react.dev/reference/react/Suspense), alongside the `LazyCountrySymbol` component. `LazyCountrySymbol` takes a `code` prop to load the appropriate Country Symbol component.
```